packages/expo-development-client/ios/EXDevelopmentClientRecentlyOpenedAppsRegistry.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import Foundation
+
+let RECENTLY_OPENED_APPS_REGISTRY_KEY = "expo.developmentclient.recentlyopenedapps"
+
+let TIME_TO_REMOVE = 60 * 60 * 24 * 3 // 3 days
+
+@objc(EXDevelopmentClientRecentlyOpenedAppsRegistry)
+public class EXDevelopmentClientRecentlyOpenedAppsRegistry : NSObject {
```

Comment:
1. I think it doesn't have to be `public` — let's keep it `internal` (default)
2. If it's not public, we don't need to prefix everything with `EXDevelopmentClient` —> it's pretty long name and makes things very specific to the dev client. I would call it just `RecentAppsRegistry` 😉